browser/src/react-redux/App/ForgotPasswordPage/ForgotPasswordPage.react.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+const initialState = {
+  email: '',
+  response: '',
+};
+
+class ForgotPasswordPage extends Component {
+  state = initialState;
+
+  handleChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value,
+    });
+  }
+
+  onSubmit = (e) => {
+    e.preventDefault();
+    this.setState({ response: '' });
+    const { email } = this.state;
+    const userEmail = {
+      email,
+    };
+    axios
+      .post('/api/auth/forgot', userEmail)
+      .then(() => {
+        this.setState({ response: 'You should receive an email, soon, with instructions to reset your password.' });
+      })
```

Comment:
I believe code in the `.then()` block will only execute on a status response of 200, but I am not 100%.

Reply:
I don't believe a response with a status of 500 will be caught as an error. Have you tested this? You can test this by changing the route to just server up the 500 status immediately when the request comes in.

Reply:
Looks to me like 500 is getting caught as an error. Let me know if it looks different on your end. Thanks!